### ISSUE TITLE
[Adaptive Card : Labels and instructions]: Screen reader does not announces the name of place when focus lands on 'James Russel' card. 

### ISSUE THREAD

**Issue by carlos-zamora** [carlos-zamora/AdaptiveCards#28]
https://microsoft.visualstudio.com/OS/_workitems/edit/33201369
